apps/dashboard/.env.example

```
@@ -8,16 +8,30 @@
 DATABASE_HOST="aws.connect.psdb.cloud"
+# "unkey" for local
 DATABASE_USERNAME=
+# "password" for local
 DATABASE_PASSWORD=
 
 # Unkey
 
+# "ws_local_root" for local
 UNKEY_WORKSPACE_ID=
+# "api_local_root_keys" for local
 UNKEY_API_ID=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The UNKEY_API_ID key should go before the UNKEY_WORKSPACE_ID key

(UnorderedKey)